docs/quickstart.md

```diff
@@ -0,0 +1,164 @@
+<!--[metadata]>
++++
+title = "Quickstart Guide"
+description = "Getting started with Docker Compose"
+keywords = ["documentation, docs,  docker, compose, orchestration, containers"]
+[menu.main]
+parent="smn_workw_compose"
+weight=3
++++
+<![end-metadata]-->
+
+
+## Quick start
```

Comment:
We've been standardizing on "Getting Started" rather than Quick start.  Can you rename the file and the heading to match with your description.